src/app/qt/facefilter/VideoFilterRunnable.hpp

```diff
@@ -37,9 +37,14 @@
 #define VIDEO_FILTER_RUNNABLE_HPP_
 
 #include <QVideoFilterRunnable>
-#include <QOpenGLFunctions> // introduce GLuint in cross-platform fashion
 
+// Includes 'glew.h' {
 #include "drishti/acf/GPUACF.h"
+// }
```

Comment:
Do you still have the "include stack" to where glew.h is getting included?  I'm curious if it can be removed or hidden in a cpp file.